mesonbuild/modules/i18n.py

```diff
@@ -27,6 +36,9 @@ def gettext(self, state, args, kwargs):
         datadirs = mesonlib.stringlistify(kwargs.get('data_dirs', []))
         extra_args = mesonlib.stringlistify(kwargs.get('args', []))
 
+        if not self.has_gettext_tools():
+            mlog.warning("Some gettext tools were not found. Installation of languages won't be available until all tools are installed")
```

Comment:
Fair enough, I will get to it, but it won't be done before monday. I was only thinking about the code structure, not the meson.build input, thanks.